### ISSUE TITLE
History of aortic valve replacement

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10031]
**Preferred term label:**
History of aortic valve replacement

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Prosthetic aortic valve degeneration

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000983
- label: History of aortic valve replacement
- parents: [History of cardiovascular procedure (HP:0034420)](https://hpo.jax.org/app/browse/term/HP:0034420)
- pmid: PMID:31354937
- def: History of having received a surgical replacement of the aortic valve.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10031
- synonyms:
